### PR TITLE
edited syntax and handled error in ThreeD_Plots.jl

### PR DESCRIPTION
**Pull request by maeckha**

### CODE REVIEW

File: examples/ThreeD_Plots.jl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 using GLMakie
-using AbstractPlotting.Colors
-AbstractPlotting.inline!(true)
+using AbstractPlotting
+AbstractPlotting.inline!(true)
```
Comment (weymouth) [inline examples/ThreeD_Plots.jl:3]: careful with changes that are not really changes.

**Review by weymouth — CHANGES_REQUESTED** [review]
I can't check this vscode stuff as I'm still using atom. Is it normal to include these as files in a repo?
Also, can you clean out that line in ThreeD_Plots.jl which is just a space.

File: examples/ThreeD_Plots.jl (hunk 1 above)
Comment (maeckha) [reply examples/ThreeD_Plots.jl:3]: I changed it because with just the `.Colors` I had a `ERROR: LoadError: UndefVarError: AbstractPlotting not defined`

**Comment by weymouth** [comment]
That line is fine. On the next line, you've added a space at the end, but
git thinks it was a change. This kind of thing makes it harder to use tools
like git blame. just a small thing.

Gabriel D Weymouth
_______________________________________________
"Computers are useless. They can only give you answers." Pablo Picasso

File: examples/ThreeD_Plots.jl (hunk 1 above)
Comment (maeckha) [reply examples/ThreeD_Plots.jl:3]: I discussed this error in a Julia forum and was told to `use` the whole package.
Concerning the vscode file, that was supposed to be ignored with `gitignore`, sorry for the confusion.

**Comment by maeckha** [comment]
space deleted now in ThreeD_Plots.jl

**Pull request merged**